### PR TITLE
Added displaying labels

### PR DESCRIPTION
**Pull request by bahmutov**
```
$ npm run demo

> git-issues@1.1.0 demo /Users/kensho/git/git-issues
> ./bin/git-issues -r https://github.com/softwarescales/git-issues

┌──┬───────────┬───────────────────────────────┬────────┐
│# │Label(s)   │Title                          │Status  │
├──┼───────────┼───────────────────────────────┼────────┤
│1 │           │Test issue Open (Do not close!)│OPEN    │
├──┼───────────┼───────────────────────────────┼────────┤
│13│enhancement│Display labels                 │OPEN    │
└──┴───────────┴───────────────────────────────┴────────┘
```

### CODE REVIEW

**Comment by bahmutov** [comment]
the le-table dependency should be upgraded - breaks on empty strings, for now coding around it

File: bin/git-issues [hunk 1]
```diff
@@ -82,12 +91,19 @@ GitIssues.fromPath(repoOption.value, statusOption.value, {
     });
 
     issues.forEach(function (cI) {
-        table.addRow([cI.number, cI.title, {
-            text: cI.state.toUpperCase()
-          , data: {
+        // TODO Table seems to break if the string is undefined or empty
+        var labels = labelsFor(cI) || " ";
```
Comment (IonicaBizau) [inline bin/git-issues]: Ohhh, I remember I did this too, lol. :smile:

**Comment by IonicaBizau** [comment]
@bahmutov Thanks! I will take a look in the next days. Make sure you create an issue in `le-table`. :joy:

**Comment by IonicaBizau** [comment]
Thanks! :sparkles: 

:ship: **`1.2.0`** :tada:

**Pull request merged**